src/main/java/org/openhds/service/impl/census/IndividualService.java

```diff
@@ -39,9 +71,45 @@ public Individual recordIndividual(Individual individual,
                                        String membershipUuid,
                                        String relationshipUuid,
                                        String fieldWorkerId) {
+
         individual.setCollectedBy(fieldWorkerService.findOrMakePlaceHolder(fieldWorkerId));
 
-        //TODO: Handle side effect creation for things like socialgroup and membership etc.
+        //Create the individual first, so that everything after is guaranteed to point at a real entity
+        individual = createOrUpdate(individual);
+
+        //TODO: Is the socialgroup created here?
+        //TODO: Do we make two paths depending on if it's a member or head of household?
+        SocialGroup socialGroup = socialGroupService.findOrMakePlaceHolder(socialGroupUuid);
+        Location location = locationService.findOrMakePlaceHolder(locationUuid);
+
+        //TODO: How are we going to start using project codes?
+        //TODO: Do these side effect entities need to call findOrMakePlaceholder?
+        /* It's my understanding that these entities are only ever created when an individual is created
+         this means that a placeholder doesn't exist yet. However, there is really no harm in calling
+         the method and overwriting whatever it returns, even if it's NOT a placeholder (e.g. some other
+         implementation supports the explicit creation of things like memberships outside of the context of
+         creating an individual... Should we be checking if the entity is return is a placeholder or not? if so, how?
+         There is currently no normalized way of checking all entities 'types' (placeholder, unknown, normal, etc). */
```

Comment:
I think placeholder and unknown should be made explicit as part of the auditable entity status.

Projects probably want to know if they have any dangling placeholders floating around.  If we make the status explicit, and resolve the status when the real record arrives, then projects will be able to do a query and report on placeholders.